integration-tests/md/extend.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: md raid /dev/md0 with partitions sdb[1-4] and unused partition sdb5
```

Comment:
Indeed, but as long as I don't have a framework where hundreds of such tests can easily be run automatically it's difficult to add code that checks the result. And I consider comments in human language as to inaccurate.